SettingsList.py

```diff
@@ -2467,6 +2461,39 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             <b>Spirit Temple</b>: lobby elevator activated,
             shortcut silver blocks moved, central room
             platform lowered, and statue face melted
+
+            Choice: Select dungeons with shortcuts
```

Comment:
Should also be choose

Reply:
Fixed :https://github.com/TestRunnerSRL/OoT-Randomizer/pull/1493/commits/a0e2952eabb59bda65bd4ee3ca66f82250fcf08a